bridge/client/sdp.js

```diff
@@ -288,13 +295,29 @@ if (typeof(SDP) == "undefined")
             if (ufrag && pwd) {
                 mediaDescription.ice = {
                     "ufrag": ufrag[1],
-                    "password": pwd[1]
+                    "password": pwd[1],
+                    "iceOptions": {}
                 };
             }
+            var iceOptions = match(mblock, regexps.iceoptions, "m", sblock);
+            if (iceOptions) {
+                var canTrickle = match(iceOptions[1], regexps.trickle);
+                if (canTrickle) {
+                    if (!mediaDescription.ice)
+                        mediaDescription.ice = {
+                            "iceOptions": {}
+                        };
```

Comment:
Even though this "if" has a single statement, it spans several lines. Please use { }.